### ISSUE TITLE
Adapt Accessibility tests: Alt text for images

### ISSUE THREAD

**Issue by gareehilsdon** [adaptlearning/adapt_framework#317]
Location:
Go to: https://community.adaptlearning.org/course/view.php?id=4
navigate with keybaord, using tab and pressing enter/space to select ['Adapt V1.1.0 - Accessibility']
Takes user to: https://community.adaptlearning.org/mod/scorm/view.php?id=43
navigate with keybaord, using tab and pressing enter/space to select [Enter]
Takes user here: https://community.adaptlearning.org/mod/scorm/player.php
navigate with keyboard, using tab and pressing enter/space to select [Adapt v1.0.1 - testing]
HOME:Welcome to Adapt Learning - v1.0.0 
https://community.adaptlearning.org/pluginfile.php/1287/mod_scorm/content/3/index.html

The three images on this page have not alt text to describe them, however as they are links too, the logical thing to do would be to name them the link that they go to. 
Images on this page need alt text, jaws says 'alt text same page link graphic.', alt text should be 'Welcome to Adapt Learning' for the first one for example.
